src/addToDoForm.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 
 
-const AddToDoForm = () => {
+const AddToDoForm = (props) => {
+    const handleAddToDo = (event) => {
+        event.preventDefault();
+        const toDoTitle= event.target.title.value;
+        console.log(toDoTitle);
+        event.target.title.value= '';
+        props.onAddToDo(toDoTitle);
+    };
     return(
-        <form>
-            <label htmlFor="Title">Title</label> <input id= "Search"></input><button>Add</button>
+        <form onSubmit={handleAddToDo}>
+            <label htmlFor="Title">Title</label> 
+            <input id= "Search" type= "text" name= "title" />
```

Comment:
proper formatting would remove the spaces here.  All markup should be like:
`type="text"`

Reply:
I see, thank you for the corrections! I will be more consistent on that going forward.